tests/odov3/odo_test.go

```diff
@@ -39,7 +39,6 @@ var stacksPath string
 var stackDirs string
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
I understand we removed this due to it being deprecated now in our updated Go version but did it serve an important purpose? Should we be looking into using the suggested change [here](https://pkg.go.dev/math/rand#Seed)?